### PR TITLE
LRDOCS-6019 and LRDOCS-6227

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-6019, https://issues.liferay.com/browse/LRDOCS-6227, hopefully both of these tickets are taken care of by this pull.

### CODE REVIEW

**Comment by lipusz** [comment]
Looks so much better ;-)

File: discover/reference/articles/06-elasticsearch-settings.markdown [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+Elasticsearch is the default search engine for @product-ver@. The *Liferay
+Foundation* suite includes an adapter for Elasticsearch called *Liferay Portal
+Search Elasticsearch*. The adapter is configurable through System Settings or an
+OSGi configuration file named
+`com.liferay.portal.search.elasticsearch.configuration.ElasticsearchConfiguration.cfg`
+and deployed to `[Liferay_Home]/osgi/configs`.
+
++$$$
+
+**Elasticsearch 6:** The [Elasticsearch 6 connector](MP LINK WHEN RELEASED)
```
Comment (lipusz) [inline discover/reference/articles/06-elasticsearch-settings.markdown]: The link can be removed as the ES6 app is part of the 7.1.x bundle. For 7.0.x, we should add the link to the app page.

**Comment by rbohl** [comment]
Sure, I'll port to 7.0 (including the link to the connector app).

**Comment by lipusz** [comment]
Nice. Let me know once you pushed the new commit and I'll forward to Rich if you agree.

**Comment by rbohl** [comment]
Pushed new commit. Thanks Tibor, you're saving my life by catching that stuff!

**Comment by lipusz** [comment]
Actually, I'm generating extra work for you... :stuck_out_tongue:

**Comment by rbohl** [comment]
Hey,  it's job security ;)

**Comment by lipusz** [comment]
Forwarded to https://github.com/sez11a/liferay-docs/pull/3878.
See my changes: https://github.com/lipusz/liferay-docs/compare/0cc042bb211637e4a0315924e6552d9572f23467..pr-23-LRDOCS-6227_LRDOCS-6018-7.1.x

Thanks! :+1:

**Pull request closed without merging**